daemons/gptp/common/ptp_message.cpp

```diff
@@ -1502,12 +1510,23 @@ void PTPMessagePathDelayRespFollowUp::processMessage(IEEE1588Port * port)
 	    Timestamp prev_peer_ts_theirs;
 	    FrequencyRatio rate_offset;
 	    if( port->getPeerOffset( prev_peer_ts_mine, prev_peer_ts_theirs )) {
+		    FrequencyRatio upper_limit, lower_limit;
+		    upper_limit = UPPER_FREQ_LIMIT*2;
+		    upper_limit /= US_PER_SEC;
+		    upper_limit += 1;
+		    lower_limit = UPPER_FREQ_LIMIT*2;
```

Comment:
Agreed that the lower_limit should be based on the LOWER_FREQ_LIMIT. The reason to add 1 is:

UPPER_LIMIT = 250 // units of PPM
upper_limit = UPPER_LIMIT/1e6 + 1 
upper_limit = 1.000250 // This is the upper limit of the neighbor rate ratio

The lower limit should be computed:
LOWER_LIMIT = -250 // Again, in PPM units
lower_limit = LOWER_LIMIT/1e6 + 1 
lower_limit = .999750 // This is the lower limit of the neighbor rate ratio.

I originally added the UPPER and LOWER limits to prevent excessive disturbances to the PI controller not the neighbor rate ratio. It seems like a fine idea to limit the neighbor rate ratio, but in reference to the comment above we should probably justify this. The code looks like maybe you want the limit to be +/- 500 PPM?